aytests/dns_setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e -x
+
+fgrep -q 'WRITE_HOSTNAME_TO_HOSTS="no"' /etc/sysconfig/network/dhcp || exit 1
```

Comment:
If I am not wrong, as you're using `set -e`, you won't need to call exit. It will stop on the first command that fails.